docs/run/primitives.mdx

```diff
@@ -90,6 +90,91 @@ Example:
 estimator.run([(circuit1, observable1, param_values1),(circuit2, observable2, param_values2) ])
```

Comment:
Tiny suggestion:

```suggestion
estimator.run([(circuit1, observable1, param_values1),(circuit2, observable2, param_values2)])
```

Reply:
I cannot leave an inline suggestion because the code wasn't changed but I was checking the rendered output and the example for EstimatorV1 seems to get cut out, it might make sense to divide it into two lines:

![Screenshot 2024-03-13 at 10 27 54](https://github.com/Qiskit/documentation/assets/57907331/30f2b1bb-06df-4743-9665-56c4c9c019d2)